packages/web-components/src/menu/menu-split.stories.ts

```diff
@@ -0,0 +1,180 @@
+import { html, repeat } from '@microsoft/fast-element';
+import { type Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
+import { ButtonAppearance, ButtonShape, ButtonSize } from '../button/button.options.js';
+import type { Menu as FluentMenu } from './menu.js';
+
+type Story = StoryObj<FluentMenu>;
+
+const defaultSlottedContent = html`
+  <fluent-menu-list style="inset-area: block-end span-inline-start;">
+    <fluent-menu-item>Menu item 1</fluent-menu-item>
+    <fluent-menu-item>Menu item 2</fluent-menu-item>
+    <fluent-menu-item>Menu item 3</fluent-menu-item>
+    <fluent-menu-item>Menu item 4</fluent-menu-item>
+  </fluent-menu-list>
+`;
+
+const generatePimaryActionSlottedContent = (content: string = 'Primary Action') => {
+  return html`<fluent-button
+    appearance="${story => story.appearance}"
+    shape="${story => story.shape}"
+    size="${story => story.size}"
+    slot="primary-action"
+    >${content}</fluent-button
+  >`;
+};
+
+const defaultTriggerSlottedContent = html`<fluent-menu-button
+  appearance="${story => story.appearance}"
+  shape="${story => story.shape}"
+  size="${story => story.size}"
+  slot="trigger"
+  icon-only
```

Comment:
Should we add `aria-label="Open menu"` here just to encourage and remind people that they should still give the button an accessible name when it's an icon-only button?